src/accelerate/commands/env.py

```diff
@@ -0,0 +1,51 @@
+import argparse
+import platform
+
+import numpy as np
+
+from accelerate import __version__ as version
+from accelerate.file_utils import is_torch_available
+
+
+def env_command_parser(subparsers=None):
+    if subparsers is not None:
+        parser = subparsers.add_parser("env")
+    else:
+        parser = argparse.ArgumentParser("Accelerate env command")
+
+    if subparsers is not None:
+        parser.set_defaults(func=env_command)
+    return parser
+
+
+def env_command(args):
+    pt_version = "not installed"
+    pt_cuda_available = "NA"
+    if is_torch_available():
+        import torch
+
+        pt_version = torch.__version__
+        pt_cuda_available = torch.cuda.is_available()
+
+    info = {
+        "`accelerate` version": version,
+        "Platform": platform.platform(),
+        "Python version": platform.python_version(),
+        "Numpy version": np.__version__,
+        "PyTorch version (GPU?)": f"{pt_version} ({pt_cuda_available})",
+    }
```

Comment:
Maybe we could add the content of the default config to this info? It can be done in a followup PR if you don't have time to add it right now.

Reply:
I don't understand, default config for what?

Reply:
The default config of Accelerate in `default_config.yaml`

Reply:
Okay, got it, I'll add this